examples/unix_socket_server/dub.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "unit-socket-server-example",
+	"description": "A minimal HTTP server using Unix domain sockets",
+	"dependencies": {
+		"vibe-d:http": {"path": "../../"},
+		"vibe-d:web": {"path": "../../"},
+		"vibe-d:core": {"path": "../../"}
```

Comment:
FWIW you would only need `vibe-d:web` -  it properly specified the dependencies on the other modules.

Reply:
Hmm, I think I copied this from another example's dub.json. Might be worth taking a look at others to see if they have unnecessary dependencies as well then.

Reply:
`vibe-d:core` is still required for the server example so the `libevent` configuration can be used for it, otherwise dub would say `The sub configuration directive "vibe-d:core" -> "libevent" references a package that is not specified as a dependency and will have no effect.`. Removing the `vibe-d:http` dependency from the server example and `vibe-d:web` from the client example though.